modules/reindex/src/main/java/org/elasticsearch/index/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -331,6 +387,7 @@ public void readFrom(StreamInput in) throws IOException {
         retryBackoffInitialTime = new TimeValue(in);
         maxRetries = in.readVInt();
         requestsPerSecond = in.readFloat();
+        workers = in.readVInt();
```

Comment:
I expect to backport this to 5.1 so I'm aware we'll need version guards here on backport.